Assets/MixedRealityToolkit/_Core/Definitions/Devices/TrackingState.cs

```diff
@@ -18,6 +18,10 @@ public enum ControllerState
         /// </summary>
         Other,
         /// <summary>
+        /// The controller is tracked, but has approximate positioning.
```

Comment:
Recommend removing this in favor of the bool in #2347 to simplify checking if a device is tracked.

Reply:
I disagree